app/src/main/res/values/dimens.xml

```diff
@@ -496,4 +496,9 @@
   <dimen name="promoted_story_card_layout_width">280dp</dimen>
   <dimen name="promoted_story_card_text_size">16sp</dimen>
   <dimen name="promoted_story_card_margin_top">8dp</dimen>
+
+  <!--Topic Practice-->
```

Comment:
<!-- Topic Practice -->

Add space before and after text inside comment. Ditto for other files too.

Reply:
Done.